src/main/java/util/DiscordHelper.java

```diff
@@ -0,0 +1,43 @@
+package util;
+
+import org.javacord.api.entity.Mentionable;
+import org.javacord.api.entity.channel.Channel;
+import org.javacord.api.entity.permission.Role;
+import org.javacord.api.entity.user.User;
+import org.javacord.api.interaction.InteractionBase;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.stream.Collectors;
+
+public class DiscordHelper {
+    /**
+     * Gets the display name of the user in a channel
+     *
+     * @param user    The user to check the nickname of
+     * @param channel The channel to check
+     * @return The display name of the user in a channel, or their name if it's not a server channel
+     */
+    public static String getUsernameInChannel(User user, Channel channel) {
```

Comment:
I have this and the last function in this file in UtilFunctions.java, so it might be better if you delete those and update the functions that call this to use your implementation.

Reply:
They were both copied from UtilFunctions. I do not think it's better to do so, as that'll end up with a LOT of extra changed files that are unnecessary to the functioning of this PR.